dev/new-rule/helpers/to-title-case.js

```diff
@@ -0,0 +1,10 @@
+function toTitleCase(newRuleName) {
```

Comment:
Should probably remove this helper file, it is not used (or update the actual script to use it)